src/Controllers/QuizController.php

```diff
@@ -98,14 +124,20 @@ public function addQuiz(RequestInterface $request)
         return $this->renderer->renderView("login.phtml", $request->getRequestParameters());
```

Comment:
I will use a redirect here when i will start the refactoring issue. Now it's not the goal of the pull request

Reply:
ok